rpm/archivematica/Makefile

```diff
@@ -0,0 +1,40 @@
+NAME          = archivematica
+VERSION       = 1.4.1
```

Comment:
Did you read https://github.com/artefactual-labs/am-packbuild/pull/9#discussion-diff-55615861 before it was closed?